src/exampleTypes/TermsStatsTable.js

```diff
@@ -19,14 +19,14 @@ let SimpleFilter = observer(({ Input = 'input', ...props }) => (
   </Flex>
 ))
 let SelectSize = observer(
-  ({ node, tree, options = [10, 25, 50, 100, 500, 1000] }) => (
+  ({ node, tree, options = [10, 20, 50, 100, 500, 1000] }) => (
```

Comment:
`options` is already a prop. We should probably just customize this where is used